### PR TITLE
test image viewer w/ fixed with

### PR DESCRIPTION
**Pull request by oded-soffrin**

### CODE REVIEW

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 1]
```diff
@@ -409,4 +407,10 @@ ImageViewer.propTypes = {
   blockKey: PropTypes.string,
 };
 
-export default ImageViewer;
+const WrapperImage = props => {
+  const newProps = {
+    ...props,
+  };
+  return <ImageViewer {...newProps} />;
```
Comment (oded-soffrin) [inline packages/plugin-image/web/src/image-viewer.jsx:414]: what's that?

Comment (justame) [reply packages/plugin-image/web/src/image-viewer.jsx:454]: temporary for testing

**Comment by justame** [comment]
@oded-soffrin @lxgreen please review the **logic** and let me know if you have any comments.

**Comment by oded-soffrin** [comment]
@justame we can close this one, right?

**Comment by justame** [comment]
Yes, sure

**Pull request closed without merging**